api.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios'
+
+function getAllArticles() {
+    return axios.get('https://news-project-1.onrender.com/api/articles').then((response)=> {
+        return response.data.articles
```

Comment:
Love that you're taking the info off the correct keys in this file rather than in your component
Also great that you've got a separate file for the api requests 🙌 
You might want to consider using the `baseURL` functionality available in axios, it will mean you'll only ever have the put the baseURL in one place in your code, notes here https://notes.northcoders.com/courses/js-front-end/axios#axios-instance